src/main/kotlin/Strategy/strategies/ReducedStrategy.kt

```diff
@@ -0,0 +1,3 @@
+package Strategy.strategies
+
+val ReducedStrategy : (Double) -> Double = { it + it * 0.10 }
```

Comment:
`it` is a lambda here.

In kotlin you can make labda funtions like this: 

```kotlin
val numbers = listOf(1,2,3,4)
numbers.map { number -> number * number } // returns [1, 4, 9, 16]
```

We can omit `number` parameter definition and call it `it` instead.

```kotlin
numbers.map { it * it }
```

In our example, `ReducedStrategy` is a **high order function** which receives a _Double_ and returns a _Double_. `it` is referencing that _Double_ parameter.